app/extensions/tus/__init__.py

```diff
@@ -95,9 +99,30 @@ def _tus_upload_file_handler(
             }
             json.dump(metadata, metadata_file)
 
-    log.debug('Tus finished uploading: {!r} in dir {!r}.'.format(filename, dir))
     filepath = os.path.join(dir, filename)
 
+    max_files = int(app.config.get('TUS_MAX_FILES_PER_TRANSACTION', 5000))
```

Comment:
why are defaults needed here AND in config.py? 
Surely the ones in config.py should be sufficient?

Reply:
Yeah I feel a bit weird about doing the defaults here as well, I'll change it